include/votca/xtp/bulkesp.h

```diff
@@ -0,0 +1,115 @@
+/* 
+ *            Copyright 2009-2017 The VOTCA Development Team
```

Comment:
2018

Reply:
Should this be done for all files or just the ones I changed in this branch?

Reply:
✓